src/components/impact/OurSolution.js

```diff
@@ -2,214 +2,123 @@ import React from 'react';
-import img6 from '../../assets/impactPage/water.png';
-import img7 from '../../assets/impactPage/energy.png';
-import img8 from '../../assets/impactPage/growth.png';
-import img9 from '../../assets/impactPage/infractructure.png';
-import img10 from '../../assets/impactPage/inequalities.png';
-import img11 from '../../assets/impactPage/communities.png';
-import img12 from '../../assets/impactPage/consuption.png';
-import img13 from '../../assets/impactPage/climate.png';
-import img14 from '../../assets/impactPage/belowWater.png';
-import img15 from '../../assets/impactPage/liveOnSand.png';
-import img16 from '../../assets/impactPage/justice.png';
-import img17 from '../../assets/impactPage/partnerships.png';
-
-const cards = [
-  {
-    title: 'No Poverty',
-    image: img1,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
-  },
-  {
-    title: 'Zero Hunger',
-    image: img2,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
-  },
-  {
-    title: 'Good Health &\nWell-being',
-    image: img3,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
-  },
-  {
-    title: 'Quality\nEducation',
-    image: img4,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
-  },
-  {
-    title: 'Gender\nEquality',
-    image: img5,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
-  },
-  {
-    title: 'Clean Water\n& Sanitation',
-    image: img6,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
-  },
-  {
-    title: 'Affordable &\nClean Energy',
-    image: img7,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
-  },
-  {
-    title: 'Decent Work &\nEconomic Growth',
-    image: img8,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
-  },
-  {
-    title: 'Industry,\nInnovation &\nInfrastructure',
-    image: img9,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
-  },
-  {
-    title: 'Reduced\nInequalities',
-    image: img10,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
-  },
+const objectifs = [
   {
-    title: 'Sustainable Cities\n& Communities',
-    image: img11,
-    textOnOver: 'Lorem ipsum dolor sit amet, lorem ipsum dolro sit emat Lorem ipsum dolor sit amet, lorem ipsum dolro',
+    iconURL: iconProvide,
+    goal: '<bold>Provide<bold> volunteering support to organizations without forcing them to cut into their budget',
```

Comment:
Does this `<bold>` render properly? I'd expect this to be a `React.Fragment`, rather than a normal string...